index.php

```diff
@@ -75,11 +78,12 @@
 				</div>
 				<label for='userKeyField' class='col-md-1 control-label auth-field'>User Key</label>
 				<div class="col-md-4">
-					<input class='form-control auth-field' name="userKeyField" type="text" id="userKeyField" value="<?php echo $userKey; ?>">
+					<input class='form-control auth-field' name="userKeyField" type="password" id="userKeyField" value="<?php echo $userKey; ?>">
 				</div>
 				<div class="col-md-1">
 					<input id='deauthBtn' class="btn btn-danger" type='button' name="authBtn" value='Deauthenticate' onclick='deAuthenticate()'>
 					<input id='manualAuthBtn' class="btn btn-primary hidden" type='button' name='authBtn' value='Save' onclick="authenticateFields()" >
+					<input id="toggle-user-key" class="btn btn-danger" type="button" value="Show" onclick="togglePasswordiness('user')"></input>
```

Comment:
I expect this would look better if it was ordered first. Would be right next to the field, and in line with the app key show/hide that way.

Reply:
Yeah, it does. Done.